auth/oidc.go

```diff
@@ -159,6 +159,9 @@ type OIDCProvider struct {
 	// Sync Gateway and the underlying OIDC library.
 	UsernameClaim string `json:"username_claim"`
 
+	RolesClaim    string `json:"roles_claim"`
```

Comment:
Can remove the whitespace on line 161.

Reply:
Huh, there's none there for me?
![image](https://user-images.githubusercontent.com/2904440/168028012-d43b5d62-b0ba-420a-b073-6cb4317785d9.png)